packages/devtools_app/analysis_options.yaml

```diff
@@ -11,6 +11,10 @@ analyzer:
     # expected.
     - test/test_infra/test_data/syntax_highlighting/**
 
+linter:
```

Comment:
should this be added to the main analysis options file instead? or is it intentional we only want this in devtools_app
https://github.com/flutter/devtools/blob/master/analysis_options.yaml

Reply:
is there a way to exclude the test directory for this print?

Reply:
When adding it at the top level it required a lot more changes, so I wanted to keep it as relevant to our logging changes as possible.

I tried really hard to find a way to exclude directories here, but unfortunately it isn't supported :( So i opted to do the per-file ignores which I know is pretty lame :(
https://github.com/dart-lang/linter/issues/320